### PR TITLE
Work on subcolumns

### PR DESCRIPTION
**Pull request by bakerac4**
This just allows us to make the sub columns visible. We still need substantial work to make it work with our visible columns array. This also temporarily exports the types that we need across projects

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -11,7 +11,7 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
-indent_size = 2
+indent_size = 4
```
Comment (billdami) [inline .editorconfig:14]: do you have latest master merged into your branch? not sure why this is showing as a diff, its already `4` on master.

File: addon/components/table.ts [hunk 2]
```diff
@@ -11,9 +11,109 @@ import { htmlSafe } from '@ember/string';
 import { SafeString } from '@ember/template/-private/handlebars';
 import Media from 'ember-responsive';
 import NativeArray from '@ember/array/-private/native-array';
-import { ColumnValue, TableSort, RowClickEvent } from '@gavant/ember-table';
+// import { ColumnValue, TableSort, RowClickEvent } from '@gavant/ember-table';
```
Comment (billdami) [inline addon/components/table.ts:14]: sine these were not externally accessible before, i think we should just remove the types from this file so we dont have them defined in 2 places, and just import them from this from if they are being used elsewhere within the addon. And then if we can figure out a better way to export them we can move them later.

File: .editorconfig (hunk 1 above)
Comment (bakerac4) [reply .editorconfig:14]: I probably don't, but its still letting you merge because there are no conflicts

File: types/ember-table/index.d.ts [hunk 3]
```diff
@@ -1,105 +1,2 @@
 declare module 'ember-table/components/ember-table/component';
 declare module 'ember-table/components/ember-tbody/component';
-declare module '@gavant/ember-table/-private/sticky/legacy-sticky-polyfill';
-declare module '@gavant/ember-table/-private/sticky/table-sticky-polyfill';
```
Comment (billdami) [inline types/ember-table/index.d.ts:4]: these 2 module declarations still need to be there as they are JS files copied over from the ember-table addon

**Pull request merged**